src/Favicon/Favicon.php

```diff
@@ -311,9 +314,13 @@ private function checkImageMType($url) {
     private function checkImageMTypeContent($content) {
         if(empty($content)) return false;
 
-        $fInfo = finfo_open(FILEINFO_MIME_TYPE);
-        $isImage = strpos(finfo_buffer($fInfo, $content), 'image') !== false;
-        finfo_close($fInfo);
+        $isImage = true;
+        try {
+            $fInfo = finfo_open(FILEINFO_MIME_TYPE);
+            $isImage = strpos(finfo_buffer($fInfo, $content), 'image') !== false;
+            finfo_close($fInfo);
+        } catch (Exception $e) {
+        }
```

Comment:
Addressed in https://github.com/ArthurHoaro/favicon/pull/5/commits/b4308ffe168f19c54b368808023f5b0bfa1b62aa
The errors occur when fileinfo appears to be installed (which is not always the case), with `extension_loaded('fileinfo')` and `function_exists('finfo_open')` returning true, and the finfo_* functions crashing when actually using them https://github.com/FreshRSS/FreshRSS/issues/1461